piperider_cli/configuration.py

```diff
@@ -36,6 +74,93 @@ def is_piperider_workspace_exist(workspace_path: str = PIPERIDER_WORKSPACE_PATH)
     return True
 
 
+class TelemetryIdResolver:
+
+    def __init__(self):
+        self.telemetry_id = None
+        self.resolvers = [
+            self.resolve_from_local_state,
+            self.resolve_from_cloud_state,
+            self.resolve_from_legacy_config,
+            self.resolve_by_new_id,
+        ]
+        self.show_debug_message = os.environ.get('SHOW_PIPERIDER_TELEMETRY_ID_RESOLVED_TIME') is not None
+
+    def resolve(self, configuration: "Configuration") -> str:
```

Comment:
by moving `class TelemetryIdResolver` behind `class Configuration`, we don't need to quote `Configuration`